duckduckgo/macos-browser
# Reset bookmarks sync timestamp after migrating to FFS favorites

Task/Issue URL: https://app.asana.com/0/0/1205710208879158/f

**Description**:
After migration to form-factor specific favorites is done, reset the timestamp
of bookmarks syncable model to force full remote data fetch to populate
FFS favorites folders that may have previously been skipped.

**Steps to test this PR**:
1. Launch the **macOS DMG app** from Xcode from develop branch.
2. Sign up to Sync.
3. Launch the **macOS App Store app** from this branch.
4. Add some favorites to the App Store app.
5. Connect App Store app to DMG app's Sync account, display form factor specific favorites (default setting).
7. Verify that both apps have the same set of favorites.
8. Delete a favorite on the DMG app. Verify that it's not deleted from the App Store app (that displays desktop favorites that were unaffected by a change made on DMG app).
9. Run DMG app from this branch.
10. Verify that DMG app displays the previously deleted favorite.



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
